### PR TITLE
dolphin: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/dolphin.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> KDE file manager.
+> More information: <https://apps.kde.org/dolphin/>.
+
+- Launch Dolphin:
+
+`dolphin`
+
+- Launch Dolphin and display a specific directory:
+
+`nautilus {{path/to/directory}}`
```
Comment (bl-ue) [inline pages/linux/dolphin.md:12]:
```suggestion
`dolphin {{path/to/directory}}`
```

File: pages/linux/dolphin.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch Dolphin with a specific file or directory selected:
+
+`dolphin --select {{path/to/file_or_directory}}`
+
+- Launch Dolphin in a separated window:
+
+`nautilus --new-window`
```
Comment (bl-ue) [inline pages/linux/dolphin.md:20]:
```suggestion
`dolphin --new-window`
```

File: pages/linux/dolphin.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch Dolphin with a split view:
+
+`dolphin --split`
+
+- Start Dolphin daemon (only required for DBus interface):
+
+`nautilus --daemon`
```
Comment (bl-ue) [inline pages/linux/dolphin.md:28]:
```suggestion
`dolphin --daemon`
```

File: pages/linux/dolphin.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display help:
+
+`nautilus --help`
```
Comment (bl-ue) [inline pages/linux/dolphin.md:32]:
```suggestion
`dolphin --help`
```

**Review by bl-ue — COMMENTED** [review]
Copy/paste mistake? 🙂

**Comment by navarroaxel** [comment]
@bl-ue They're too similar so I've cloned the nautilus page.

File: pages/linux/dolphin.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch Dolphin in a separated window:
+
+`dolphin --new-window`
+
+- Launch Dolphin with a split view:
+
+`dolphin --split`
+
+- Start Dolphin daemon (only required for DBus interface):
```
Comment (bl-ue) [inline pages/linux/dolphin.md:26]:
```suggestion
- Start the Dolphin daemon (only required to use the DBus interface):
```

File: pages/linux/dolphin.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch Dolphin with a specific file or directory selected:
+
+`dolphin --select {{path/to/file_or_directory}}`
+
+- Launch Dolphin in a separated window:
+
+`dolphin --new-window`
+
+- Launch Dolphin with a split view:
```
Comment (marchersimon) [inline pages/linux/dolphin.md:22]:
```suggestion
- Launch Dolphin in split view:
```
Does this sound better?

Comment (bl-ue) [reply pages/linux/dolphin.md:22]: Yeah, I like it better 👍🏻

**Review by marchersimon — APPROVED** [review]

**Review by Waples — APPROVED** [review]

**Pull request merged**